Dunno if you want to fix it, there are some errors with Windows_cl [starting here](https://dev.azure.com/martinkroeker/martinkroeker/_build/results?buildId=1181&view=logs&j=2d8ea0da-d599-5234-e06f-743200a8aa8e&t=30994467-7236-5890-7de5-bc1a1b096700&l=9907)
```
utest\test_gemv.c(22,15): error C2057: expected constant expression
```

I think that is coming from declarations like this https://github.com/OpenMathLib/OpenBLAS/blob/fb7c53c5e56469d95bcf13345bc3532381873fb6/utest/test_gemv.c#L21-L23

where N is not a constant expression.

--- a/utest/test_gemv.c
+++ b/utest/test_gemv.c
@@ -7,13 +7,14 @@
 #ifndef INFINITY
 #define INFINITY 1.0/0.0
 #endif
+#define N 17
+#define Ny 33
 
 #ifdef BUILD_SINGLE
 
 CTEST(sgemv, 0_nan_inf)
 {
     int i;
-    blasint N = 17;
     blasint incX = 1;
     blasint incY = 1;
     float alpha = 0.0;
@@ -39,8 +40,6 @@ CTEST(sgemv, 0_nan_inf)
 CTEST(sgemv, 0_nan_inf_incy_2)
 {
     int i;
-    blasint N  = 17;
-    blasint Ny = 33;
     blasint incX = 1;
     blasint incY = 2;
     float alpha = 0.0;
@@ -73,7 +72,6 @@ CTEST(sgemv, 0_nan_inf_incy_2)
 CTEST(dgemv, 0_nan_inf)
 {
     int i;
-    blasint N = 17;
     blasint incX = 1;
     blasint incY = 1;
     double alpha = 0.0;
@@ -99,8 +97,6 @@ CTEST(dgemv, 0_nan_inf)
 CTEST(dgemv, 0_nan_inf_incy_2)
 {
     int i;
-    blasint N  = 17;
-    blasint Ny = 33;
     blasint incX = 1;
     blasint incY = 2;
     double alpha = 0.0;

--- a/utest/test_gemv.c
+++ b/utest/test_gemv.c
@@ -7,22 +7,21 @@
 #ifndef INFINITY
 #define INFINITY 1.0/0.0
 #endif
-#define N 17
-#define Ny 33
 
 #ifdef BUILD_SINGLE
 
 CTEST(sgemv, 0_nan_inf)
 {
     int i;
+    blasint N = 17;
     blasint incX = 1;
     blasint incY = 1;
     float alpha = 0.0;
     float beta = 0.0;
     char  trans = 'N';
-    float A[N * N];
-    float X[N];
-    float Y[N];
+    float A[17 * 17];
+    float X[17];
+    float Y[17];
 
     memset(A, 0, sizeof(A));
     memset(X, 0, sizeof(X));
@@ -40,14 +39,16 @@ CTEST(sgemv, 0_nan_inf)
 CTEST(sgemv, 0_nan_inf_incy_2)
 {
     int i;
+    blasint N = 17;
+    blasint Ny = 33;
     blasint incX = 1;
     blasint incY = 2;
     float alpha = 0.0;
     float beta = 0.0;
     char  trans = 'N';
-    float A[N * N];
-    float X[N];
-    float Y[Ny];
+    float A[17 * 17];
+    float X[17];
+    float Y[33];
     float *ay = Y;
 
     memset(A, 0, sizeof(A));
@@ -72,14 +73,15 @@ CTEST(sgemv, 0_nan_inf_incy_2)
 CTEST(dgemv, 0_nan_inf)
 {
     int i;
+    blasint N = 17;
     blasint incX = 1;
     blasint incY = 1;
     double alpha = 0.0;
     double beta = 0.0;
     char  trans = 'N';
-    double A[N * N];
-    double X[N];
-    double Y[N];
+    double A[17 * 17];
+    double X[17];
+    double Y[17];
 
     memset(A, 0, sizeof(A));
     memset(X, 0, sizeof(X));
@@ -97,14 +99,16 @@ CTEST(dgemv, 0_nan_inf)
 CTEST(dgemv, 0_nan_inf_incy_2)
 {
     int i;
+    blasint N = 17;
+    blasint Ny = 33;
     blasint incX = 1;
     blasint incY = 2;
     double alpha = 0.0;
     double beta = 0.0;
     char  trans = 'N';
-    double A[N * N];
-    double X[N];
-    double Y[Ny];
+    double A[17 * 17];
+    double X[17];
+    double Y[33];
     double *ay = Y;
 
     memset(A, 0, sizeof(A));

